src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicEditAndContinue.cs

```diff
@@ -209,7 +209,7 @@ End Module
 ");
             VisualStudio.Workspace.WaitForAsyncOperations(FeatureAttribute.Workspace);
             VisualStudio.Debugger.Go(waitForBreakMode: false);
-            VisualStudio.ActivateMainWindow(skipAttachingThreads: true);
```

Comment:
Why was attaching threads a problem?

Reply:
Agreed, can we update the commit description to explain if we think this is ready?

Reply:
Unfortunately it is not something I have the ability to explain.